compose.yaml

```diff
@@ -0,0 +1,14 @@
+services:
+  proxy:
```

Comment:
Should we add `dummy` (which will later be `api_gateway`) to the `depends_on` list of this service, perhaps?